base/MAPL_LocStreamMod.F90

```diff
@@ -666,6 +735,11 @@ subroutine MAPL_LocStreamCreateFromFile(LocStream, LAYOUT, FILENAME, NAME, MASK,
           call MAPL_CommsBcast(vm, DATA=STREAM%TILING(N)%IM, N=1, ROOT=0, RC=status)
           call MAPL_CommsBcast(vm, DATA=STREAM%TILING(N)%JM, N=1, ROOT=0, RC=status)
        enddo
+       if (use_pfaf_) then
+          STREAM%TILING(2)%IM = stream%pfafstetter_catchments
+          STREAM%TILING(2)%JM = 1
+          STREAM%TILING(2)%name = "ROUTE_GRID"
+       end if
 
```

Comment:
Can we call the new grid "Catchment grid", or "Pfafstetter grid" insead of "ROUTE GRID"